website/static/js/osfLanguage.js

```diff
@@ -82,6 +82,8 @@ module.exports = {
         dataListFetchError: 'Could not load list of developer applications at this time. ' + REFRESH_OR_SUPPORT,
         dataSendError: 'Error sending data to the server: check that all fields are valid, or contact ' + SUPPORT_LINK + ' if the problem persists.',
         creationSuccess: 'Successfully registered new application',
-        dataUpdated: 'Application data updated'
+        dataUpdated: 'Application data updated',
+        resetSecretConfirm: 'Are you sure you want to reset this secret key? This will invalidate the application using it and revoke all access tokens. This cannot be reversed.',
```

Comment:
I think we should improve this wording: resetting the secret doesn't *invalidate* the application. The application can still be used, but it will need to do so with the new value of client secret.

(also, avoid the "key" terminology- the OAuth2 spec widely refers to "client ID" and "client secret". We should avoid inventing alternate language for such common terms)

Applies here and for template text below, in https://github.com/CenterForOpenScience/osf.io/pull/4441/files#diff-22948d17c254cc633034b2fa091d2049R44

I propose the following wording:
> 'Are you sure you want to reset the client secret? This cannot be reversed. Your application will be unusable until it is updated with the new client secret, and all users must reauthorize access. Previously issued access tokens will no longer work.'